docs/plugins/pluggable_rule_engine.md

```diff
@@ -243,6 +243,7 @@ remote("host","hints") {
 "hints" (required) are of the form:
 
   - `ZONE` - Remote Zone - The name of the Zone in which the "host" is located.
+  - `INST_NAME` - Instance Name - The name of the rule instance to target on the remote host.
```

Comment:
"of the rule instance" -> "of the Rule Engine Plugin instance"

Reply:
Yes, that's better.